### PR TITLE
Make GPG key optional

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Fixes http://ci.theforeman.org/job/release_test_katello_nightly_el7/305/artifact/debug/fb-install-katello.bats.tap/*view*/

File: manifests/init.pp [hunk 1]
```diff
@@ -75,7 +75,7 @@
 #                       type:Boolean
 #
 # $repo_gpgkey::        The GPG key to use
-#                       type:String
+#                       type:String[Optional]
```
Comment (stbenjam) [inline manifests/init.pp]: Other way, `Optional[String]`

Comment (ehelms) [inline manifests/init.pp]: Doh, need a kafo linter.

**Comment by ehelms** [comment]
Updated

**Pull request merged**